src/ui/components/Rating/index.js

```diff
@@ -68,9 +68,12 @@ export class RatingBase extends React.Component {
         id: `Rating-rating-${thisRating}`,
         key: `rating-${thisRating}`,
         ref: (ref) => { this.ratingElements[thisRating] = ref; },
+        title: (rating && !readOnly) ? i18n.sprintf(i18n.gettext(`Update your rating to %(thisRating)s out of 5.`), { thisRating }) :
```

Comment:
Looking at this code, if  `!rating && readOnly === true` then we will show "Rate this add-on %(thisRating)s out of 5.", which is not intended.

Did you test this out and get the desired result? This `readOnly` check seems especially superfluous given you set `props.title` to `null` later on.